pkg/config/default.go

```diff
@@ -540,11 +542,6 @@ func (c *Config) Env() []string {
 	return c.Containers.Env
 }
 
-// InitPath returns location where init program added to containers when users specify the --init flag.
```

Comment:
This would cause a breaking change, why not still return the init-path?

Reply:
Because this is a broken API, nobody should call that, the correct logic is in FindInintBinary(). Returning a single path just causes confusion for users who think this function does return something correct. 
We do not guarantee a stable API in c/common so I can just nuke it here and update the only caller in podman.